wings/1.0/installing.md

```diff
@@ -140,6 +140,16 @@ sudo wings --debug
 
 You may optionally add the `--debug` flag to run Wings in debug mode.
 
+### Configuring DNS (Usually not needed, refer back to this if your containers have no internet connection)
+Now that Wings has run successfully and you have gotten the green heart on your Nodes page, the wings config at '/etc/pterodactyl/config.yml' will have new values.
+One of those values is DNS, which by default will be 1.1.1.1 and 1.0.0.1
+For your containers to have internet, your DNS Servers in your wings config MUST match your DNS Servers on your host system.
+To view what DNS Servers your host system is using, you can use systemd-resolve, which typically will be on any system that uses systemd.
```

Comment:
You should specify that `systemd-resolve --status` is the command to run if they want to see their DNS servers.

It's also important to know that not all distributions use `systemd-resolve` to manage their dns.